packages/core/src/storybook/components/canvas-wrapper/CanvasWrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import { Canvas, useOf } from "@storybook/blocks";
+import { ComponentProps, FC, useMemo, useState } from "react";
+import styles from "./CanvasWrapper.module.scss";
+
+type CanvasWrapper = ComponentProps<typeof Canvas>;
+
+const CanvasWrapper: FC<CanvasWrapper> = ({ of }) => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  const { story } = useOf(of || "story", ["story"]);
+
+  const toggleCodeAction = useMemo(
+    () => ({
+      title: "Toggle code editor",
```

Comment:
Suggestion: call it "Play"

Reply:
Just like that? Play?
It opens and shut the live editor, not sure about Play